cpp/src/aztec/stdlib/hash/pedersen/pedersen.test.cpp

```diff
@@ -280,39 +280,6 @@ template <typename Composer> class stdlib_pedersen : public testing::Test {
         EXPECT_EQ(result, true);
     }
 
-    static void test_pedersen_large_unrolled()
```

Comment:
`test_pedersen_large` remains and  tests the unrolled case (the only case now!).